src/BenchmarkDotNet/Toolchains/DotNetCli/DotNetCliExecutor.cs

```diff
@@ -21,6 +22,7 @@ public class DotNetCliExecutor : IExecutor
         public DotNetCliExecutor(string customDotNetCliPath) => CustomDotNetCliPath = customDotNetCliPath;
 
         private string CustomDotNetCliPath { get; }
+        private readonly TimeSpan ProcessExitTimeout = TimeSpan.FromSeconds(2);
```

Comment:
since it's more or less a const, we can make it `static`

```suggestion
        private static readonly TimeSpan ProcessExitTimeout = TimeSpan.FromSeconds(2);
```